wgsl/index.bs

```diff
@@ -9338,7 +9379,13 @@ notations:
 
           Note: This is a [=partial assignment=] to *x*.
 
-          Note: Partial assignments include the previous value since only a subset of components are updated.
+          Note: Partial assignments include the previous value since,
+          in the non-[=degenerate partial reference=] case,
+          only a subset of components are updated.
+
+          Note: An assignment through a [=degenerate partial reference=] is analyzed
+          as if it modifies all the locations in the
+          associated [=originating variable=].
```

Comment:
You're missing an inversion here. Above you say it is analyzed as if not modifying all memory locations.

Reply:
fixed in next commit.